architect.yml

```diff
@@ -1,32 +1,36 @@
 name: nuxt
-description: A starter project showing how to make use of JSX with Typescript in Nuxt.js
+description: A starter project showing how to make use of Typescript in Nuxt.js
 keywords:
   - nuxt.js
   - typescript
-  - jsx
-  - tsx
+  - vue
+
+dependencies:
+  node-rest-api: latest
```

Comment:
We want to make sure that this repo can be pulled down as a stand-alone project. We can add in a dependency later, but for a first touch, the user should be able to run `architect dev` on the project without having to pull down anything additional.

Here's an example of the [React project](https://github.com/architect-templates/react/pull/10) that I did. I included the lightweight-api so there is some resemblance of a dynamic website, but we want to wait till a bit later to introduce dependencies

Reply:
This is also important for the UI - if there's a dependency, the project will fail because it doesn't register both components